purchase_variant_configurator_on_confirm/models/purchase_order_line.py

```diff
@@ -44,20 +42,21 @@ def onchange_product_tmpl_id(self):
 
         fpos = self.order_id.fiscal_position_id
         company_id = self.env.user.company_id.id
-        self.taxes_id = fpos.map_tax(
-            self.product_tmpl_id.supplier_taxes_id.filtered(
-                lambda r: r.company_id.id == company_id)
-        )
+        if self.env.uid == SUPERUSER_ID:
```

Comment:
Why this unfolding? It's incorrect, as other users apart from admin can have access to several companies (through a parent company)